### PR TITLE
Improve bug template info

### PR DESCRIPTION
**Pull request by leticiafernandes**
Improve github bug template with more infos and attention points.

### CODE REVIEW

**Review by joaopaulovieira — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -16,7 +16,13 @@ labels: bug
 * then second step
 * I was expecting X but instead it shows Y
 
-Did you try to reproduce this issue at http://cdn.clappr.io/
+**Is this only happenning in your streaming?**
```
Comment (leandromoreira) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]: Does it happen only in your streaming?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -16,7 +16,13 @@ labels: bug
...
+* If yes, please provide to us the public video URL
```
Comment (leandromoreira) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: `If it does, please provide your streaming URL.`

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (leticiafernandes) [reply .github/ISSUE_TEMPLATE/bug_report.md]: @leandromoreira Thank you for the syntax improvements hints.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (leticiafernandes) [reply .github/ISSUE_TEMPLATE/bug_report.md]: @leandromoreira Thank you for the syntax improvements hints.

**Review by pedrochamberlain — APPROVED** [review]

**Pull request merged**